src/UrlQueryReader.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\DataView;
+
+final class UrlQueryReader
+{
+    public function get(string $name): ?string
+    {
+        $value = $_GET[$name] ?? null;
```

Comment:
Shouldn't we get it from PSR request instead of `$_GET`?

Reply:
For use request we need somewhere find it.

1) Pass request to widget via special method. It's very not convenient.

2) Create `RequestProvider` and catch request sowewhere. Same implementation we use in Yii Input HTTP: https://github.com/yiisoft/input-http?tab=readme-ov-file#general-usage

But what is problem `$_GET` usage?

Reply:
Now I remove `UrlQueryReader` in favor `UrlParameterProviderInterface` usage.